src/Model/FundingAccount.php

```diff
@@ -0,0 +1,352 @@
+<?php
+namespace CurrencyCloud\Model;
+
+use DateTime;
+
+class FundingAccount implements EntityInterface {
+
+    /**
+     * @var string
+     */
+    private $id;
+    /**
+     * @var string
+     */
+    private $accountId;
+    /**
+     * @var string
+     */
+    private $accountNumber;
+    /**
+     * @var string
+     */
+    private $accountNumberType;
+    /**
+     * @var string
+     */
+    private $accountHolderName;
+    /**
+     * @var string
+     */
+    private $bankName;
+    /**
+     * @var string
+     */
+    private $bankAddress;
+    /**
+     * @var string
+     */
+    private $bankCountry;
+    /**
+     * @var string
+     */
+    private $currency;
+    /**
+     * @var string
+     */
+    private $paymentType;
+    /**
+     * @var string
+     */
+    private $regularRoutingCode;
+    /**
+     * @var string
+     */
+    private $regularRoutingCodeType;
+    /**
+     * @var string
+     */
+    private $priorityRoutingCode;
+    /**
+     * @var string
+     */
+    private $priorityRoutingCodeType;
+    /**
+     * @var DateTime
+     */
+    private $createdAt;
+    /**
+     * @var DateTime
+     */
+    private $updatedAt;
+
+    /**
+     * @return string
+     */
+    public function getId()
```

Comment:
missing return types (whole class)

Reply:
I agree, but it's convention of this sdk. I think it's php 5.4 compatible.